
int8 NOCHANGE = -1
int8 IDLE = 0
int8 EXPLORE = 1
int8 GO_TO_TARGET = 2
int8 ANALYZE_TARGET = 3

int8 behavior
medos_demo/Target target

